sponsorship_compassion/models/res_partner.py

```diff
@@ -1,250 +1,263 @@
-# -*- coding: utf-8 -*-

```

Comment:
Maybe an encoding issue? Please check why the whole file are rewrittent. It's not good for keeping track of the code changes...

Reply:
The encoding issue seems to be back on this file...